DALI.tex

```diff
@@ -700,31 +717,23 @@ \subsection{Intervals}
 
 \noindent are all legal integer interval values.
 
-Floating point interval values serialised in VOTable or service parameters must have the following metadata in the
-\xmlel{FIELD} element: \verb|datatype="double"| or \verb|datatype="float"|,
-\verb|arraysize="2"|, \verb|xtype="interval"|.
-
-Integer interval values serialised in VOTable or service parameters must have the following metadata in the
-\xmlel{FIELD} element: \verb|datatype="short"| or \verb|datatype="int"| or
-\verb|datatype="long"|, \verb|arraysize="2"|, \verb|xtype="interval"|.
-
-Interval values serialised in VOTable (\xmlel{FIELD}) or service parameters
-(\xmlel{PARAM}) with this xtype may include additional metadata like minimum
+Interval values serialised in VOTable or described in described in service parameters
+with this xtype may include additional metadata like minimum
 or maximum value. These are specified using the standard scalar \verb|MIN| and
 \verb|MAX| child elements to describe the (minimum) lower bound and (maximum)
 upper bound of interval(s) respectively.
 
 \subsection{Sexagesimal Coordinates}
-Coordinate values expressed in sexagesimal form can be described using the following
-xtypes in both VOTable \xmlel{FIELD} and \xmlel{PARAM} elements:
-
-\begin{itemize}
-\item right ascension: \verb|datatype="char"| \verb|arraysize="*"| \verb|xtype="hms"|
-\item declination: \verb|datatype="char"| \verb|arraysize="*"| \verb|xtype="dms"|
-\end{itemize}
+Coordinate values expressed in sexagesimal form can be described using the 
+following VOTable type metadata. For right ascension:
 
-\noindent
-(the arraysize may also be a fixed length or variable length with limit).
+\begin{verbatim}
+datatype="char" arraysize="*" xtype="hms"
+\end{verbatim}
+\noindent and for declination:
+\begin{verbatim}
+datatype="char" arraysize="*" xtype="dms"
+\end{verbatim}
 
 For \verb|xtype="hms"|, the values are serialised as hours:minutes:seconds where hours
 and minutes are integer values and seconds is a real value. For \verb|xtype="dms"|, the values
```

Comment:
Sexagesimal strings usually represent (arc)minutes and the integer part of (arc)seconds as 2-digit numbers with a leading zero if required.  The existing text doesn't explicitly allow this, and most of the examples (e.g. "`90:0:0`") don't do it.  I suggest adding text to explicitly allow leading zeros, and editing the examples to use leading zeros where appropriate (e.g. "`90:00:00`").

(The text could encourage or even require 2-character fields with leading zeros for minute/second fields below 10 but it gets a bit fiddly, since maybe hours and even degrees should be so padded as well, so this is probably more complication than warranted).